internal/zinc-core/src/main/scala/sbt/internal/inc/InvalidationProfiler.scala

```diff
@@ -11,8 +11,7 @@
 
 package sbt.internal.inc
 
-import sbt.internal.prof.Zprof
-
+// import sbt.internal.prof.Zprof
```

Comment:
Did you mean to leave all of this in as comments?

Reply:
Yea, I'm just going to comment out the protobuf usage, and later figure out what to do with the InvalidationProfiler instead of outright deleting it, if that's ok.